internal/tasks/storage.go

```diff
@@ -82,6 +82,10 @@ func (j *Janitor) sweepStorage(ctx context.Context, account models.Account, _ pr
 	if err != nil {
 		return err
 	}
+	err = j.sweepTrivyReportStorage(ctx, reducedAccount, actualTrivyReports, canBeDeletedAt)
+	if err != nil {
+		return err
+	}
```

Comment:
Note to self: I do not have test coverage for this part yet. I put up the PR regardless so everyone can start reviewing the other parts.